app/api/trace/route.ts

```diff
@@ -43,6 +45,7 @@ export async function POST(req: NextRequest) {
       { status: 200 }
     );
   } catch (error) {
+    console.error(error);
```

Comment:
possibly remove if this was just for testing

Reply:
done